### DISCUSSION TITLE
Is there a way to organize filters in different rows?

### DISCUSSION THREAD

**Discussion by ntraykov** [rappasoft/laravel-livewire-tables#1039] [Q&A] [answered]
I couldn't find that in the documentation. Thanks in advance!

**Comment by lrljoe** [comment]
I'm assuming you're referring to the slide down rather than popover.

Unfortunately not at present, I am currently working on numerous changes to how filtering works, and rows of filters for slide down is on the list.

**Reply by lrljoe** [reply]
I've got a proof of concept working for this, so you can specify the row that the filter should appear in.  As always, it'll be a bit until I can issue a PR due to having to write the tests & documentation for it.

I'm assuming the attached is what you're talking about btw, so just the Number Range filter has Row 3 specified.
![Filter - Determined Rows](https://user-images.githubusercontent.com/104938042/224577131-9ef5f6bc-0678-416b-b9a2-8f2372f33945.png)

**Reply by ntraykov** [reply]
Yes, that is exactly what I was asking for! Thank you very much!

**Reply by lrljoe** [reply]
In 2.12, due for release imminently

**Comment by lrljoe** [comment, marked as answer]
I figured that adding in Column Span setting would also be useful for this:
![Filter - Determined Rows - Colspan](https://user-images.githubusercontent.com/104938042/224578053-dbc16ef2-5252-4e1a-a37b-d8f33034d9e9.png)

Let me know thoughts on the above, if it meets your needs then I'll write some tests/docs over the next couple of days

**Reply by ntraykov** [reply]
That is even better! Great job! Thank you!

**Comment by lrljoe** [comment]
I've opened PR #1101 - Just so you know I haven't forgotten about this.

I do still need to write the tests, but figured you'd be able to at least see what I've done in case you want to replicate it yourself

**Comment by dgillier** [comment]
Hi,
Great news !

@lrljoe  I'm curious about the range number widget shown on your screenshot, did you code something specific ?

**Reply by lrljoe** [reply]
Well spotted, I did indeed, it's available as part of [this package](https://github.com/LowerRockLabs/LaravelLivewireTablesAdvancedFilters), which is current semi-pre-release, as indicated by the compatibility matrix on the page, feel free to give it a test though and let me know any feedback.

**Reply by dgillier** [reply]
Will do !
Thanks ++

**Comment by lrljoe** [comment]
Quick update, just the Visuals test left to write, and then it'll just be a wait for it to be reviewed by the repo owner.

I do have a large number of different PRs open, so it may be a while before he has a chance to review certain ones, if this is an important feature for you, then I'd suggest you go onto the PR and say so!

**Comment by lrljoe** [comment]
Now in #1111

**Comment by lrljoe** [comment]
Available in [v2.12.0](https://github.com/rappasoft/laravel-livewire-tables/releases/tag/v2.12.0)

**Reply by ntraykov** [reply]
Thank you very much!
